jupyterlab_widgets/src/semvercache.ts

```diff
@@ -28,6 +28,12 @@ export class SemVerCache<T> {
     }
   }
 
+  getAllModules(key: string): Record<string, any> | undefined {
```

Comment:
Can we change this to `getAllVersions`? I think that would be a bit clearer about what is being returned.